### ISSUE TITLE
Check existence of $HOME/bin

### ISSUE THREAD

**Issue by jklmnn** [ytomino/headmaster#7]
The Makefile doesn't check the existence of `~/bin`.
If you call `make -C main install BINDIR=$HOME/bin` (without trailing slash) it will create a binary called `bin` in your home directory or if you call it with trailing slash it will fail to install the binary.
Afaik if you call `install -D` instead of `install` in your install target it should generate the correct behaviour.

**Comment by ytomino** [comment]
Oh, Thanks. I didn't notice it. 
And I checked now, `install` of my Mac don't have `-D` option.
So I will change it to `install -d $(EXENAME) $(BINDIR)/$(EXENAME)` instead.

Also, it does not copy `man1/headmaster.1` as of now.
However, I guess that most people don't want to create `$HOME/man1` or `$HOME/share/man1`...???
As the root of the problem, "`make -C main install BINDIR=$HOME/bin`" in readme perhaps is wrong. It should be rewritten to use `PREFIX`.

**Comment by jklmnn** [comment]
Well I don't want to have `man` or `share/man` in my home ;)
I don't know the exact behaviour of `install -d` but from the manpage it seems to be sufficient.

**Comment by ytomino** [comment]
Ah...I had read mistake about `install -d`. It does not copy files.
Finally, I added `mkdir -p`, as 6fd7f5df.

> Well I don't want to have man or share/man in my home ;)

Indeed.

Since it's a good opportunity, I separated `make install-bin BINDIR=...` (for copying only the command) from `make install PREFIX=...` (for the full-installation with copying `man1/headmaster.1`) and rewrote readme, on the same commit.

Thanks.
